mozilla-release/mobile/android/base/java/org/mozilla/gecko/preferences/PreferenceManager.java

```diff
@@ -105,4 +105,23 @@ public boolean isAutocompleteEnabled() {
     public boolean isBackgroundEnabled() {
         return  mAppSharedPreferences.getBoolean(GeckoPreferences.PREFS_CLIQZ_TAB_BACKGROUND_ENABLED,true);
     }
+
+    public boolean showCustomizeTabView() {
```

Comment:
rename it to isCustomizeTabViewShown()
Otherwise I will understand that you will show it